protelis/protelis-test/src/test/resources/method08.pt

```diff
@@ -0,0 +1,10 @@
+// EXPECTED_RESULT: 18
+
+import java.lang.String.format
+import java.util.Arrays.asList
+
+// Check call of over-length and minimum-length varargs method
+let l = asList(5,3,7,2,1); // Check call of varargs with multiple args
```

Comment:
OK, I hav switched to Lists.newArrayList and removed the security kludge.  I think this should be ready now.